### ISSUE TITLE
Persist messages after js redirect

### ISSUE THREAD

**Issue by aland** [maximebf/php-debugbar#285]
I am redirecting some pages using `top.location` in javascript, but this unfortunately clears any debugbar message I'd previously added. I thought `stackData()` may help but cannot get it to work.
Is it possible to use `stackData()` with javascript redirects, or is there another way?

**Comment by barryvdh** [comment]
I don't think so. You can add storage, so that you can inspect previous request (using the browse button), but not automatically. Can't you redirect with PHP?

**Comment by aland** [comment]
Possibly, I'm working on an existing codebase and can't get PHP redirects working immediately (server error but nothing is logged). I'll have a look at storage.
